itest/loadtest/load_test.go

```diff
@@ -55,6 +77,31 @@ func TestPerformance(t *testing.T) {
 		if !success {
 			t.Fatalf("test case %v failed", tc.name)
 		}
+
+		// Calculate the test duration and push metrics if the test case succeeded.
+		if cfg.PrometheusGateway.Enabled {
+			duration := time.Since(startTime).Seconds()
+
+			// Update the metric with the test duration.
+			testDuration.WithLabelValues(tc.name).Set(duration)
+
+			// Create a new pusher to push the metrics.
+			pushURL := cfg.PrometheusGateway.Host + ":" +
+				strconv.Itoa(cfg.PrometheusGateway.Port)
+
+			pusher := push.New(pushURL, "load_test").
+				Collector(testDuration).
+				Grouping("test_case", tc.name)
```

Comment:
Does the test name include factors such as the number of assets created, number before/after, etc? If not, we may want to add additional metrics, but as new time series rather than labels (don't want the labels to get too long).

Reply:
The label was updated so that each gauge gets a unique one, in order to distinguish the pushed metrics. On promQL it's fairly easy to aggregate everything into a single time series regardless of being individual gauges

The assets minted / sent are configured in the `loadtest.conf` file so that number is always stable (for now)

We can rely on the tapd prom metrics to relate loadtest durations with tapd behavior